deploy.sh

```diff
@@ -1,63 +0,0 @@
-#!/bin/bash -xe
```

Comment:
+1 to Ben's comment, I think we should not delete this until prod javabuilder is fully switched over to the pipeline version. Otherwise we can't update prod right?

Reply:
Correct. There is risk of configuration drift, as the pipeline replicates some of the configuration and work done in the deploy script.

I'll undelete it, and make sure they're in-sync as of this PR (taking the script as the source of truth)